### ISSUE TITLE
Not handling color names correctly.

### ISSUE THREAD

**Issue by M1ndo** [tecosaur/engrave-faces#14]
I've been debugging for hours trying to figure out why org latex export doesn't not fully compile and fail and it turns out engrave-faces does not handle hex and color names correctly when exporting to latex.
 i have the following source block that contains something like.
![image](https://user-images.githubusercontent.com/44820142/210281229-58227e6c-add8-40a3-8c85-3b438f7e964e.png)
Anything before line will be compiled and not after it
```lisp
 (doom-blend 'orange 'green size-index)
 (doom-blend 'red 'orange (- size-index 1)
```
exporting to latex will result into something like this.
![image](https://user-images.githubusercontent.com/44820142/210281431-2028034d-155e-42fe-96ab-c24496e3a056.png)

Also hex colors are not correctly set.
![image](https://user-images.githubusercontent.com/44820142/210281519-f4906491-2736-44b9-a998-751f5d095f94.png)
![image](https://user-images.githubusercontent.com/44820142/210281541-ea4f5c3a-d0d3-4438-8422-3c385aaaf41f.png)

Update: Tried this PR and it fixed the issue [360cbc0](https://github.com/tecosaur/engrave-faces/pull/13/commits/360cbc0bd73b8944cb848a78265ffd8cb63a34c3)

**Comment by tecosaur** [comment]
Thanks for the issue! I assume this is due to the rainbow highlighting of `green`/`red`, yes?

**Comment by M1ndo** [comment]
I think all color names are having the same issue, not just `green`/`red`.

**Comment by tecosaur** [comment]
This should be fixed in e541cf43665965e13b0629b609a14058de6153ad.
